apps/studio/lib/error-reporting.ts

```diff
@@ -77,31 +77,33 @@ function handleResponseError(error: ResponseError, context: string) {
     // Only capture 5XX errors as critical errors
     captureMessage({
       context,
-      message: `[CRITICAL][${context}] Failed requestPathname: ${requestPathname}: ${message}`,
+      message: `requestPathname ${requestPathname} w/ message: ${message}`,
     })
     return
   }
 }
 
 function handleError(error: Error, context: string) {
-  const errorMessage = error.message
-  if (!errorMessage) {
+  const message = error.message
+  if (!message) {
     return
   }
 
   captureMessage({
-    message: `[CRITICAL][${context}] Failed: ${errorMessage}`,
+    message,
     context,
   })
 }
 
 function handleUnknownError(error: unknown, context: string) {
-  const hasMessage =
-    error && typeof error === 'object' && 'message' in error && typeof error.message === 'string'
-
-  if (hasMessage) {
+  if (
+    error &&
+    typeof error === 'object' &&
+    'message' in error &&
+    typeof error.message === 'string'
+  ) {
     captureMessage({
-      message: `[CRITICAL][${context}] Failed: ${error.message}`,
+      message: error.message,
       context,
```

Comment:
Normally I hate this, but Typescript doesn't safely determine error.message is a string if its assigned :(